code/game/machinery/firealarm.dm

```diff
@@ -124,7 +124,7 @@
 	last_alarm = world.time
 	var/area/A = get_area(src)
 	A.firealert(src)
-	playsound(src.loc, 'goon/sound/machinery/FireAlarm.ogg', 75)
+	playsound(src.loc, 'sound/machines/firealarm.ogg', 75)

```

Comment:
Remove the old sound from the goon folder and change over the only other use of this sound as well